### PR TITLE
Fix websockets web platform tests

### PR DESCRIPTION
**Pull request by sherryzy**
b/332367155

### CODE REVIEW

**Review by johnxwork — COMMENTED** [review]
Great find! The wpt server's handling code could be this: http://shortn/_m7uNJrMDP3 Likely the wpt server changes the response file type or connection type when "pipe=sub" is provided. Something's not working and we are very close to the issue now, but we could debug that after the merge.

File: third_party/web_platform_tests/websockets/binary/001.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <title>WebSockets: Send/Receive blob, blob size less than network array buffer</title>
 <script src=/resources/testharness.js></script>
 <script src=/resources/testharnessreport.js></script>
-<script src=../constants.js?pipe=sub></script>
+<script src=../constants.js></script>
```
Comment (andrewsavage1) [inline third_party/web_platform_tests/websockets/binary/001.html:5]: What does having/not having the pipe=sub mean?

Comment (sherryzy) [reply third_party/web_platform_tests/websockets/binary/001.html:5]: I'm not so sure, but see John's comments above. I just found it's changed in upstream here: https://chromium-review.googlesource.com/c/chromium/src/+/2848645 and it resolves the timeout issue. I'm still trying to understand the root cause, but we can figure it out later after the merge?

**Comment by sherryzy** [comment]
This may not be the correct fix. The real issue is currently version of wpt server doesn't support HTTP2, which cause those tests timeout. To fix the issue, we have to update web_platform_tests repo to a later version which requires a much larger effort. Close this fix for now.

**Pull request closed without merging**